modules/upi-provisioning-new-hosts-in-a-upi-cluster.adoc

```diff
@@ -68,10 +178,34 @@ $ oc create -f bmh.yaml
 .Example output
 [source,terminal]
 ----
-secret/worker1-bmc created
-baremetalhost.metal3.io/worker1 created
+secret/openshift-worker-<num>-network-config-secret created
+secret/openshift-worker-<num>-bmc-secret created
+baremetalhost.metal3.io/openshift-worker-<num> created
 ----
++
+Where `<num>` is the worker number.
 
+. Power on and inspect the bare-metal node:
++
+[source,terminal]
+----
+$ oc -n openshift-machine-api get bmh openshift-worker-<num>
+----
++
+Where `<num>` is the worker node number.
++
+.Example output
+[source,terminal]
+----
+NAME                    STATE       CONSUMER   ONLINE   ERROR
+openshift-worker-<num>  available              true
```

Comment:
Available state would only happen if either the customDeploy or userData section is missing in the yaml while creating the bmh. It also happens while [Removing hosts from a user-provisioned cluster by using the BMO](https://77264--ocpdocs-pr.netlify.app/openshift-enterprise/latest/installing/installing_bare_metal/scaling-a-user-provisioned-cluster-with-the-bare-metal-operator.html#upi-removing-hosts-from-a-upi-cluster_scaling-a-user-provisioned-cluster-with-the-bare-metal-operator) 
Step 3 should delete the bmh that is now in available state
`oc delete bmh -n openshift-machine-api <bmh_name>`

Step 4 will be delete node

In step 2
update the oc edit bmh command to run on  `<bmh_name>` as well

Reply:
Updated As Suggested.